ucsb-cs156-s22/s22-6pm-happycows
# s22-6pm-3 Created Leaderboard page and tests

# Overview

Replaced the placeholder page with the leaderboard page. This includes tests to make sure the leaderboard table and page works as intended. 

Pull request also includes leaderboardFixtures.js and leaderboardTable.js, which is part of a different pull request, in order to implement the leaderboard page. 
# Test Plan (for interactive testing)

* Load the page onto Heroku/LocalHost (make sure database is clear)
* Create a commons and join with at least 2 different users (try to buy cows on 1 of the users so the leaderboard has different values for the other two users)
* add /leaderboard/(commonsId) to the url of heroku/localhost (ex: localhost:8080/leaderboard/1) to see the leaderboard (try going on admin and regular user for both leaderboards)
![image](https://user-images.githubusercontent.com/72825220/172144828-2332120f-2020-40a7-a435-ad66cc65c34a.png)
(example image)
The results will look like this,  
# Issues Addressed

Closes #9 

# Details
Test Coverage:
![image](https://user-images.githubusercontent.com/57011302/171744647-ab26a144-19e3-4afc-8f02-b3a0d591002e.png)

Mutation Tests:
![image](https://user-images.githubusercontent.com/57011302/171745855-066d98ff-0e95-4d95-a679-2db04ae7b1e0.png)
